gradle/play-publisher.gradle

```diff
@@ -0,0 +1,7 @@
+apply plugin: 'com.github.triplet.play'
+
+play {
+    track = findProperty("releaseTrack") as String ?: "internal"
```

Comment:
maybe throw exception?

Reply:
I would keep it default internal. this app will not going for production and only for internal testing.